src/styles/layouts/_layout-sign-up.scss

```diff
@@ -0,0 +1,45 @@
+.layout-sign-up {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: rem(40px) 1fr rem(200px);
+  grid-template-areas:
+    'header'
+    'main'
+    'footer';
+
+  height: 100vh;
+
+  @include respond-above('md') {
+    grid-template-rows: rem(80px) 1fr rem(250px);
+  }
+
+  @include respond-above('lg') {
+    grid-template-rows: rem(100px) 1fr rem(300px);
+  }
+
+  @include respond-above('xl') {
+    grid-template-rows: rem(140px) 1fr rem(350px);
+  }
+
+  & > header {
+    grid-area: header;
+  }
+
+  & > main {
+    grid-area: main;
+
```

Comment:
Same as https://github.com/Cobda/cobda-web/pull/7#discussion_r565484247

Reply:
It's a good practice for adding a new line as to mention we are doing separately in block-level for grid section not in cells level.